README.md

```diff
@@ -290,17 +290,18 @@ Architecture decision records start [here](doc/architecture/decisions/0001-use-a
 
-| Term  | Definition                                                                                          |
-|-------|-----------------------------------------------------------------------------------------------------|
-| GDS   | Government Digital Service - https://www.gov.uk/government/organisations/government-digital-service |
-| GDX   | Government Data Exchange - This project and the wider programme of work                             |
-| DWP   | Department for Work and Pensions                                                                    |
-| LEN   | Life Event Notification (a service from HMPO)                                                       |
-| HMPO  | HM Passport Office                                                                                  |
-| HMPPS | HM Prison and Probation Service, and executive agency of the MoJ                                    |
-| MOJ   | Ministry of Justice                                                                                 |
-| GRO   | General Registry Office                                                                             |
-| OGD   | Other Government Department                                                                         |
-| TUO   | Tell us once - https://www.gov.uk/after-a-death/organisations-you-need-to-contact-and-tell-us-once  |
-
+| Term     | Definition                                                                                          |
+|----------|-----------------------------------------------------------------------------------------------------|
+| GDS      | Government Digital Service - https://www.gov.uk/government/organisations/government-digital-service |
+| GDX      | Government Data Exchange - This project and the wider programme of work                             |
+| DWP      | Department for Work and Pensions                                                                    |
+| LEN      | Life Event Notification (a service from HMPO)                                                       |
+| HMPO     | HM Passport Office                                                                                  |
+| HMPPS    | HM Prison and Probation Service, and executive agency of the MoJ                                    |
+| MOJ      | Ministry of Justice                                                                                 |
+| GRO      | General Registry Office                                                                             |
+| OGD      | Other Government Department                                                                         |
+| TUO      | Tell us once - https://www.gov.uk/after-a-death/organisations-you-need-to-contact-and-tell-us-once  |
+| Acquirer | A downstream department or service consuming events from our API (for example DWP)                  |
+|Supplier| An upstream department or service GDX consumes events and data from (for example HMPO)              |
```

Comment:
IntelliJ will reformat this for you

Reply:
It did reformat it, but clearly failed at doing it well...